CompositeUi/src/views/page/organization/Show.js

```diff
@@ -23,13 +23,26 @@ import InlineLink from './../../component/InlineLink';
 import LoadingSpinner from './../../component/LoadingSpinner';
 import PageHeader from './../../component/PageHeader';
 import {Row, RowColumn} from './../../component/Grid';
+import AddMemberToOrganization from '../organization/AddMemberToOrganization';
 import SectionHeader from './../../component/SectionHeader';
 import Thumbnail from './../../component/Thumbnail';
 import ContentMiddleLayout from './../../layout/ContentMiddleLayout';
+import ContentRightLayout from './../../layout/ContentRightLayout';
+import CurrentOrganizationActions from './../../../actions/CurrentOrganization';
 import UserCard from './../../component/UserCard';
 
 @connect(state => ({currentOrganization: state.currentOrganization}))
 class Show extends React.Component {
+  state = {
```

Comment:
Avoid the use of the state explicitly. We are using redux so, you can dispatch an action and update the app state from reducer